benefits/core/templates/core/includes/form.html

```diff
@@ -68,19 +79,23 @@
 
       <script>
         function recaptchaSubmit($event) {
-          // Checks the validity of the form. Return if invalid; HTML5 validation errors should display
-          if (!$event.target.form.checkValidity()) {
+          // checks the validity of the form. Return if invalid; HTML5 validation errors should display
+          if (!$event.currentTarget.form.checkValidity()) {
             return;
           }
           // form is client-side valid; taking over the remainder of processing
           $event.preventDefault();
 
           grecaptcha.ready(function() {
             grecaptcha.execute("{{ request.recaptcha.site_key }}", { action: "submit" }).then(function(token) {
-                // Add the token to hidden form element
-                $event.target.form.elements["{{ request.recaptcha.data_field }}"].value = token;
+                // add the token to hidden form element
+                $event.currentTarget.form.elements["{{ request.recaptcha.data_field }}"].value = token;
+                // trigger the custom "submitting" event
+                // since calling .submit() does not trigger the "submit" event
+                // hence the normal handler won't apply
+                $($event.currentTarget.form).trigger("submitting");
```

Comment:
This solution works for me. Just curious: what was the reasoning for triggering a custom event instead of calling `requestSubmit()` on line 98 to trigger a `submit` event?

Reply:
Sorry, I missed your suggestion to use that! Been a little scattered this afternoon... I like this better. Fixing.

Reply:
Rebased to drop the old implementation and added [`cf7f670`](https://github.com/cal-itp/benefits/pull/1088/commits/cf7f670df923ef0b8221bf34841d1a1391c9f180)